Maquetacion/scss/tech.scss

```diff
@@ -0,0 +1,35 @@
+.tech {
+  background-color: $green;
+  border-radius: 5px;
+  box-shadow: -10px -10px $white, -10px -10px 0px 1px $green;
+  display: flex;
+  flex-direction: column;
+  height: 320px;
+  margin: 5px 150px;
+  text-align: center;
+  width: 220px;
+
+  .logo {
+    margin: 15px auto;
+    width: $tech-logo-size;
+  }
+
+  .title-card {
+    font-size: $text-medium;
+  }
+
+  .age {
+    font-weight: bold;
+    font-size: $text-medium;
+  }
+
+  .text {
+    margin: 10px auto;
```

Comment:
same

Reply:
the 'margin' css is removed, the 'padding' is modified